.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": ["@kth/eslint-config-kth"]
+  "extends": ["plugin:react-hooks/recommended", "@kth/eslint-config-kth"],
+  "ignorePatterns": ["public/js/lib/firebug-lite"]
```

Comment:
What do we use firebug-lite for? IIRC we removed some remnants of it in at least one repo. We should look into that in a separate task.

Reply:
I have no idea what we use it for. I considered removing it, but I ended up just adding it as a topic for our dev meetings in confluence https://confluence.sys.kth.se/confluence/display/TFI/UtbildningsInfo.